Volha_Ivanova/Module_1_Java/Classes_and_Objects/src/com/epam/cdp/volha_ivanova/task1/sweets/candies/chocolateCandies/ChocolateCandies.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.cdp.volha_ivanova.task1.sweets.candies.chocolateCandies;
+
+import com.epam.cdp.volha_ivanova.task1.sweets.candies.Candies;
+
+public class ChocolateCandies extends Candies {
```

Comment:
I understand that Candies classes represents collection of candies.
I don't recommend do that. 
I have several questions:
1) how can I add 1 chocolate milk candy additionally?
2) what I should do if my milk chocolate candy I should separate to some types, by filling for example?

I think if you choose approach when candy class will be represent 1 candy, but Gift will be collection of all the candies it will be easier.